app/src/main/res/values/strings.xml

```diff
@@ -10,196 +10,196 @@
     <string name="system_id" translatable="false">System ID</string>
     <string name="app_license" translatable="false">PCAPdroid is distributed in the hope that it will be useful, but WITHOUT
         ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-        See the <a href='https://www.gnu.org/licenses/gpl-3.0-standalone.html'>GNU General Public License</a> for
+        See the <a href='https://www.gnu.org/licenses/gpl-3.0-standalone.html'>GNU General Public License version 3 or later</a> for
         more details.</string>
-    <string name="opensource_licenses" translatable="false"><b>Open Source Licenses:</b><br/>\n\n
-        - zdtun: <a href='https://github.com/emanuele-f/zdtun/blob/master/COPYING'>GPL-3.0</a>\n\n
+    <string name="opensource_licenses" translatable="false"><b>Libre libraries and resources:</b><br/>\n\n
+        - Zero Dep Tunnel: <a href='https://github.com/emanuele-f/zdtun/blob/master/COPYING'>GPL-3.0</a>\n\n
         - nDPI: <a href='https://github.com/ntop/nDPI/blob/dev/COPYING'>LGPL-3.0</a>\n\n
-        - nanohttpd: <a href='https://github.com/NanoHttpd/nanohttpd/blob/master/LICENSE.md'>BSD-3-Clause</a>\n\n
+        - NanoHTTPD: <a href='https://github.com/NanoHttpd/nanohttpd/blob/master/LICENSE.md'>BSD-3-Clause</a>\n\n
         - CustomActivityOnCrash: <a href='https://github.com/Ereza/CustomActivityOnCrash/blob/master/LICENSE'>Apache-2.0</a>\n\n
-        - gson: <a href='https://github.com/google/gson/blob/master/LICENSE'>Apache-2.0</a>\n\n
+        - Gson: <a href='https://github.com/google/gson/blob/master/LICENSE'>Apache-2.0</a>\n\n
         - MaxMind DB Reader: <a href='https://github.com/maxmind/MaxMind-DB-Reader-java/blob/main/LICENSE'>Apache-2.0</a>\n\n
-        - Android-Flag-Kit: <a href='https://github.com/madebybowtie/FlagKit/blob/master/LICENSE'>MIT</a>\n\n
+        - FlagKit: <a href='https://github.com/madebybowtie/FlagKit/blob/master/LICENSE'>MIT</a>\n\n
         - IP Geolocation by <a href='https://db-ip.com'>DB-IP</a>\n\n
-        - Font Awesome: <a href='https://fontawesome.com/license/free'>License</a>\n\n
-        - App icon made by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">flaticon</a>\n\n
+        - Font Awesome: <a href='https://fontawesome.com/license/free'>Licenses</a>\n\n
+        - App icon by <a href="https://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">Flaticon</a>\n\n
         </string>
 
     <!-- Localized -->
     <string name="start_button">Start</string>
     <string name="stop_button">Stop</string>
     <string name="title_activity_settings">Settings</string>
-    <string name="collector_info">UDP Collector: %1$s:%2$d</string>
-    <string name="http_server_status">HTTP Server: http://%1$s:%2$d</string>
+    <string name="collector_info">UDP collector: %1$s:%2$d</string>
```

Comment:
After a while I gave up on selectively removing title case
https://material.io/archive/guidelines/style/writing.html#writing-capitalization-punctuation 
https://www.everyinteraction.com/articles/title-case-vs-sentence-case-in-ui/ and so on